### ISSUE TITLE
Build Failure after flutter upgrade

### ISSUE THREAD

**Issue by Hoax-slayer** [flutter/flutter#60847]
I am new to Flutter, working on a project, everything was working fine before but since I upgraded flutter there are loads of errors and warnings. 

Note: D:\Desktop\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.6\android\src\main\java\io\flutter\plugins\firebase\cloudfirestore\CloudFirestorePlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

D/FlutterActivity(23668): Using the launch theme as normal theme.
D/FlutterActivityAndFragmentDelegate(23668): Setting up FlutterEngine.
D/FlutterActivityAndFragmentDelegate(23668): No preferred FlutterEngine was provided. Creating a new FlutterEngine for this FlutterFragment.
D/FlutterActivityAndFragmentDelegate(23668): Attaching FlutterEngine to the Activity that owns this Fragment.
D/FlutterView(23668): Attaching to a FlutterEngine: io.flutter.embedding.engine.FlutterEngine@a6c57d8
D/FlutterActivityAndFragmentDelegate(23668): Executing Dart entrypoint: main, and sending initial route: /
Debug service listening on ws://127.0.0.1:58779/OulyD3UVTj0=/ws
Syncing files to device Redmi 3S...
I/Choreographer(23668): Skipped 45 frames!  The application may be doing too much work on its main thread.
W/DynamiteModule(23668): Local module descriptor class for providerinstaller not found.
I/DynamiteModule(23668): Considering local module providerinstaller:0 and remote module providerinstaller:0
W/ProviderInstaller(23668): Failed to load providerinstaller module: No acceptable module found. Local version is 0 and remote version is 0.
W/ResourceType(23668): No package identifier when getting name for resource number 0x00000000
V/NativeCrypto(23668): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 286 native methods...
I/art     (23668): Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.org.conscrypt.Java7ExtendedSSLSession>
I/art     (23668): Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.org.conscrypt.Java7ExtendedSSLSession>
I/art     (23668): Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.org.conscrypt.Java8ExtendedSSLSession>
I/art     (23668): Rejecting re-init on previously-failed class java.lang.Class<com.google.android.gms.org.conscrypt.Java8ExtendedSSLSession>
I/ProviderInstaller(23668): Installed default security provider GmsCore_OpenSSL

**Comment by darshankawar** [comment]
Hi @Hoax-slayer,
After upgrading, did you run `flutter clean` before trying to run the app ? Its always recommended to run `flutter clean` everytime you upgrade flutter version, in order to clear cache and pull all dependencies properly.

**Comment by Hoax-slayer** [comment]
@darshankawar Yes I did run `flutter clean` before trying to run the app. But can you tell me about these warnings as my application is running just fine it's just that since I have upgraded the flutter version and the android studio it is throwing me these statements. Also I don't know why I am getting this `I/Choreographer(23668): Skipped 45 frames! The application may be doing too much work on its main thread.`  as in earlier flutter version it didn't use to flash this warning.

**Comment by darshankawar** [comment]
@Hoax-slayer, are you still getting the build failure as you posted earlier or has that error gone now ? 

Regarding your other question on warnings, are you performing any heavy animations or db processing  or may be using large size image / resources in your app ? I'd suggest you open a new issues to better tracking from our end and if your original issue has been resolved, you may close this ticket. 
Thanks.

**Comment by Hoax-slayer** [comment]
@darshankawar no I am not getting any build failures now just these warnings, so I will close this issue. 
In my application I have used 1 or 2 SVG images whose size are also not that large, there is no such animations in the application and as far as db is concerned there are a couple of stream and future builders operations for queries. Besides I have also used shared preference but there also I am not storing a large size of data just primitive data.
